### PR TITLE
fix non android gradle instructions

### LINKED ISSUE #466: Instructions for plain gradle incorrect
plain gradle instructions are incorrect, they lead to this

given

```kt
plugins {
  `java-library`
  `java-test-fixtures`
  id("brix.bom")
  id("net.ltgt.errorprone")
  id("org.checkerframework")
}

dependencies {
  errorprone("com.google.errorprone:error_prone_core:2.+")
  annotationProcessor("com.uber.nullaway:nullaway:0.8.+")
  checkerFramework(checker.processor)
  compileOnly(checker.annotations)
  testFixturesCompileOnly(checker.annotations)
}

tasks.withType<JavaCompile>().configureEach {
  options.compilerArgs.addAll(
    listOf(
      "-parameters",
      "-Xlint:deprecation",
      "-Xlint:unchecked"
    )
  )
  options.errorprone {
    disableWarningsInGeneratedCode.set(true)
    excludedPaths.set(".*/build/generated/sources/annotationProcessor/.*")
    option("NullAway:AnnotatedPackages", "com.xenoterracide")
    val errors = mutableListOf(
    errors.add("NullAway")
    error(*errors.toTypedArray())
    ...
```

I get this error

```
> Task :util:compileTestFixturesJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':util:compileTestFixturesJava'.
> NullAway is not a valid checker name

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 5s
7 actionable tasks: 7 executed
```

### PR DESCRIPTION
**Pull request by xenoterracide**
resolves #466

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,15 +23,17 @@ plugins {
 }
 
 dependencies {
-  annotationProcessor "com.uber.nullaway:nullaway:0.9.1"
+  errorprone("com.uber.nullaway:nullaway:0.9.1")
```
Comment (msridhar) [inline README.md:26]: Should we also update the suggested version of `net.ltgt.errorprone` while we are at it?  I noticed the latest version is 2.0.1, so maybe 0.6 is quite old now

Comment (msridhar) [reply README.md:26]: Also, lower down where it says "the `annotationProcessor` line loads NullAway", that should also be changed to "the first `errorprone` line" or something like that

File: README.md [hunk 2]
```diff
@@ -73,7 +75,7 @@ Snapshots of the development version are available in [Sonatype's snapshots repo
 
 The configuration for an Android project is very similar to the Java case, with one key difference: The `com.google.code.findbugs:jsr305:3.0.2` dependency can be removed; you can use the `android.support.annotation.Nullable` annotation from the Android Support library.
 
-```gradle
+```
```
Comment (msridhar) [inline README.md:78]: why this change?

**Review by msridhar — CHANGES_REQUESTED** [review]
Thanks for this!  Couple comments

File: README.md (hunk 2 above)
Comment (xenoterracide) [reply README.md:78]: you know, I don't actually know how this happened, I'm questioning whether githubs web editor removed it, as an unrecognized extension, really it should probably be `groovy` or `kts`

File: README.md (hunk 1 above)
Comment (xenoterracide) [reply README.md:26]: yeah, that version is ancient, honestly I didn't really notice, or worry about fixing that, but maybe that's why this was written this way. Maybe the better change would just be to point to the plugin's documentation as it actually covers all of this...

**Comment by msridhar** [comment]
Sorry for not circling back to this PR for a bit.  Fixing it up requires a bit of thought and I haven't had time.  I do plan to circle back when I have more cycles, hopefully in the next couple of weeks or so

**Comment by msridhar** [comment]
@xenoterracide I finally got around to fixing up the docs in #843.  Sorry for the huge delay, and thanks again for the contribution here.
